source/js/viewer-core.js

```diff
@@ -1073,7 +1073,7 @@ export default class ViewerCore
         this.viewerState.outerObject.appendChild(errorElement);
 
         //bind dialog close button
-        document.querySelector(this.settings.selector + 'error-close').addEventListener('click', () =>
+        document.querySelector('#' + this.settings.selector + 'error-close').addEventListener('click', () =>
```

Comment:
this should probably be `document.getElementById(this.settings.selector + 'error-close')`

Reply:
`document.querySelector()` was in the existing code and didn't work with the given selector. 

I can change it to `getElementById()` assuming that the selector will be an id.